infrastructure/Pulumi.production.yaml

```diff
@@ -1,6 +1,8 @@
 config:
   aws:region: us-west-2
   www.pulumi.com:certificateArn: arn:aws:acm:us-east-1:058607598222:certificate/a3f72a2b-e715-4639-b126-1e4efc0b634b
+  www.pulumi.com:originBucketNameOverride:
```

Comment:
Since there is no value defined for this property, can this be removed? It looks like you are handling an `undefined` value for it anyway.

Reply:
It _can_ be, yes but I chose to explicitly show it, just to be clear that it's there and available.